CHANGELOG.md

```diff
@@ -5,9 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.17.0](https://github.com/Substra/substra-tools/releases/tag/0.17.0) - 2022-09-19
+
 ## Unreleased
```

Comment:
Unreleased should be on top
Have there been changes since the last release? If yes, please add them to the changelog, otherwise we do not need a release